src/Commands/ProjectCreateCommand.php

```diff
@@ -89,6 +91,24 @@ protected function execute(InputInterface $input, OutputInterface $output): int
       }
     }
 
+    $project_disk_size = $input->getArgument('project_disk_size');
+
+    if (empty($project_disk_size)) {
+      $project_disk_size = $io->ask('Please provide a PlatformSH disk size for the project (default 8 GB)');
+      if (empty($project_disk_size)) {
```

Comment:
I'd check the input provided is an int (or empty) and not something like '8000 MB'

Reply:
That's done too